src/providers/BackgroundReplacementProvider/index.tsx

```diff
@@ -95,10 +95,9 @@ const BackgroundReplacementProvider: FC<Props> = ({
   );
 
   useEffect(() => {
-    initializeBackgroundReplacement();
```

Comment:
this was never being used anyways - we always create a new one when we create a new device